### ISSUE TITLE
Why Is the Training Loss Not Decreasing After Several Epochs?

### ISSUE THREAD

**Issue by 1064815343** [LianpoWang/WLKDCA-Net#23]
During training, the loss remains almost constant or decreases extremely slowly, even after many epochs.
